runtime/doc/options.txt

```diff
@@ -772,14 +772,11 @@ A jump table for the options with a short description can be found at |Q_op|.
 	putting a ":gui" command in the gvimrc file, before where the value
 	of 'background' is used (e.g., before ":syntax on").
 
-	For Windows the default is "dark".
-	For other systems "dark" is used when 'term' is "linux",
-	"screen.linux", "cygwin" or "putty", or $COLORFGBG suggests a dark
-	background.  Otherwise the default is "light".
+	For Windows the default is "dark". Otherwise the default is "light".
```

Comment:
Yes, but we can restore the documentation on the logic at that point. When I checked how we could implement that when working on setting 'dark' as the default, I didn't find a way to do it, and I suspect that might take a while.